src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -2392,6 +2392,18 @@ namespace HVACUnitarySystem {
                 UnitarySystem(UnitarySysNum).MaxNoCoolHeatAirVolFlow / UnitarySystem(UnitarySysNum).MaxHeatAirVolFlow;
         }
 
+        // initialize idle air flow rate variables in case these are needed in multi-speed heating coil sizing when multi-speed cooling coil does not exist
+        // the multi-speed coils will overwrite this data and these variablse are only used for multi-speed coils in function SetOnOffMassFlowRate
+        if (MultiOrVarSpeedCoolCoil(UnitarySysNum) || MultiOrVarSpeedHeatCoil(UnitarySysNum)) {
+            if (UnitarySystem(UnitarySysNum).DesignFanVolFlowRate > 0.0) {
+                UnitarySystem(UnitarySysNum).IdleVolumeAirRate = UnitarySystem(UnitarySysNum).DesignFanVolFlowRate;
+            } else {
+                UnitarySystem(UnitarySysNum).IdleVolumeAirRate = max(UnitarySystem(UnitarySysNum).MaxCoolAirVolFlow, UnitarySystem(UnitarySysNum).MaxHeatAirVolFlow);
+            }
+            UnitarySystem(UnitarySysNum).IdleMassFlowRate = UnitarySystem(UnitarySysNum).IdleVolumeAirRate * StdRhoAir;
+            UnitarySystem(UnitarySysNum).IdleSpeedRatio = 1.0;
+        }
+
```

Comment:
OK, so why do this? If there is no cooling coil then the idle flow rate won't be set and will likely be 0. If a mu;ti-speed heating coil is present then the idle flow rate will be 0 when it is time to take the min of 0 and the heating flow rate. See heating coil idle flow rate set below.

Reply:
I chose the higher flow rate because of the min(heatingflow, coolingflow) calcs below and I didn't want this default to collide with "current" logic (i.e., idle flow is not based on no load flow). Not saying I'm right, your wrong, just saying that's what I did. I think you have a good point. Only VS and MS coils use that variable, but when implemented we already had a no load flow rate variable. I can't stand using 2 different variables based on coil type. I recall messing around with idle flow before and got diff's. Maybe a new issue to just focus on idle flow rate usage?